cfme/configure/configuration/diagnostics_settings.py

```diff
@@ -381,7 +381,7 @@ def is_displayed(self):
             # compare with the selection in accordion as it can be a slave server
             self.title.text ==
             self.form_expected_title(name=name,
-                                     sid=sid,
+                                     sid=int(sid[1]),
```

Comment:
can you add an inline comment why do we need to convert to `int` and select the second element?

Reply:
The `form_expected_title` collecting `name`, `sid`, and `current`. But here it is sending `sid` in '[1]' format. At the end we are getting '[[1]]' which is failing to assert. By converting it into `int` solved the problem.

Reply:
thanks, what I meant is: can you add a small inline comment in the code?

Reply:
Updated.